resources/frameworks_2021Q3.yaml

```
@@ -0,0 +1,91 @@
+---
+
+#########################
+### AutoML frameworks ###
+#########################
+
+AutoGluon:
+  version: "0.3.1"
```

If we plan not to run the default AG, I think we can use:
```suggestion
AutoGluon:
  abstract: true
  version: '0.3.1'
```